src/core/thread.d

```diff
@@ -2576,6 +2576,21 @@ else
                 __asm("stm  $0, {r4-r11}", "r", regs.ptr);
                 sp = __asm!(void*)("mov $0, sp", "=r");
             }
+            else version (MIPS32)
+            {
+                import ldc.llvmasm;
+
+                // Callee-save registers, according to MIPS Calling Convention
+                size_t[8] regs = void;
+                __asm(`sw $$0,  0($0);
```

Comment:
IIRC, this should either be `$$16` (etc., like for MIPS64), or `$s0` (etc.).

Reply:
I think the code must be framed with `.set noat` .. `.set at` to avoid warnings from assembly stage.

Reply:
@JohanEngelen  I fixed the register names, was using the wrong ones indeed.

@redstar I tested the `set .noat/at` framing and there are no more warnings.

Thanks!